### PR TITLE
Adding CardView attrs

### PR DESCRIPTION
**Pull request by charlesx2013**
Adding `radius`, `elevation`, `contentPadding` and `backgroundColor` as CardView attributes to `CardViewProxy`

### CODE REVIEW

File: paris/src/main/res/values/attrs.xml [hunk 1]
```diff
@@ -88,4 +88,10 @@
         <attr name="android:textAppearance" />
     </declare-styleable>
 
+    <declare-styleable name="Paris_CardView">
+        <attr name="cardCornerRadius" />
+        <attr name="cardElevation" />
+        <attr name="cardBackgroundColor" />
```
Comment (shaishavgandhi05) [inline paris/src/main/res/values/attrs.xml]: Can you make it alphabetical? `cardBackgroundColor` would be first.

File: paris/src/main/java/com/airbnb/paris/proxies/CardProxy.kt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import com.airbnb.paris.R2
+import com.airbnb.paris.annotations.Attr
+import com.airbnb.paris.annotations.LayoutDimension
+import com.airbnb.paris.annotations.Styleable
+
+@Styleable("Paris_CardView")
+class CardViewProxy(view: CardView) : BaseProxy<CardViewProxy, CardView>(view) {
+
+    @Attr(R2.styleable.Paris_CardView_cardCornerRadius)
+    fun setRadius(@LayoutDimension radius: Int) {
```
Comment (shaishavgandhi05) [inline paris/src/main/java/com/airbnb/paris/proxies/CardProxy.kt]: Why not float?

File: paris/src/main/java/com/airbnb/paris/proxies/CardProxy.kt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    @Attr(R2.styleable.Paris_CardView_cardElevation)
+    fun setElevation(@LayoutDimension elevation: Int) {
+        view.cardElevation = elevation.toFloat()
+    }
+
+    @Attr(R2.styleable.Paris_CardView_contentPadding)
+    fun setContentPadding(@LayoutDimension contentPadding: Int) {
+        view.setContentPadding(contentPadding, contentPadding, contentPadding, contentPadding)
+    }
+}
```
Comment (shaishavgandhi05) [inline paris/src/main/java/com/airbnb/paris/proxies/CardProxy.kt]: Endline

File: paris/src/main/java/com/airbnb/paris/proxies/CardProxy.kt [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        view.setCardBackgroundColor(colorStateList)
+    }
+
+    @Attr(R2.styleable.Paris_CardView_cardElevation)
+    fun setElevation(@LayoutDimension elevation: Int) {
+        view.cardElevation = elevation.toFloat()
+    }
+
+    @Attr(R2.styleable.Paris_CardView_contentPadding)
+    fun setContentPadding(@LayoutDimension contentPadding: Int) {
```
Comment (shaishavgandhi05) [inline paris/src/main/java/com/airbnb/paris/proxies/CardProxy.kt]: Should we provide all the 4 values? Padding isn't necessarily uniform always.

**Comment by shaishavgandhi05** [comment]
Oops didn't mean to approve. Just meant to comment.

File: paris/src/main/java/com/airbnb/paris/proxies/CardProxy.kt (hunk 4 above)
Comment (charlesx2013) [reply paris/src/main/java/com/airbnb/paris/proxies/CardProxy.kt]: we can add those are separate attributes, adding just this for now.

File: paris-cardview/build.gradle [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    testImplementation deps.mockitoCore
+    testImplementation deps.robolectric
+
+    kaptTest project(':paris-processor')
+
+    androidTestImplementation deps.mockitoAndroid
+    androidTestImplementation deps.espresso
+}
+
+apply from: rootProject.file('gradle/gradle-maven-push.gradle')
```
Comment (shaishavgandhi05) [inline paris-cardview/build.gradle]: Endline

File: paris-cardview/gradle.properties [hunk 6]
```diff
@@ -0,0 +1,3 @@
+POM_NAME=Paris cardview
+POM_ARTIFACT_ID=paris-cardview
+POM_PACKAGING=jar
```
Comment (shaishavgandhi05) [inline paris-cardview/gradle.properties]: Endline

File: paris-cardview/src/main/res/values/attrs.xml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="Paris_CardView">
+        <attr name="cardBackgroundColor" />
+        <attr name="cardCornerRadius" />
+        <attr name="cardElevation" />
+        <attr name="contentPadding" />
+    </declare-styleable>
+</resources>
```
Comment (shaishavgandhi05) [inline paris-cardview/src/main/res/values/attrs.xml]: Endline

File: paris/build.gradle [hunk 8]
```diff
@@ -26,6 +26,7 @@ android {
 
 dependencies {
     implementation deps.appcompat
+    implementation deps.cardView
```
Comment (shaishavgandhi05) [inline paris/build.gradle]: Not required anymore right?

File: paris-cardview/src/main/java/com/airbnb/paris/cardview/CardViewProxy.kt [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+        view.radius = radius
+    }
+
+    @Attr(R2.styleable.Paris_CardView_cardBackgroundColor)
+    fun setBackgroundColor(colorStateList: ColorStateList?) {
+        view.setCardBackgroundColor(colorStateList)
+    }
+
+    @Attr(R2.styleable.Paris_CardView_cardElevation)
+    fun setElevation(@Px elevation: Float) {
```
Comment (shaishavgandhi05) [inline paris-cardview/src/main/java/com/airbnb/paris/cardview/CardViewProxy.kt]: Maybe `setCardElevation`? `CardView` extends `View` which already has a `setElevation`. I'm not sure if the implementations are different but best to mirror the property you're mutating IMO.

**Pull request closed without merging**